storage/src/main/scala/ch/epfl/bluebrain/nexus/storage/StorageError.scala

```diff
@@ -95,6 +96,11 @@ object StorageError {
     */
   final case class OperationTimedOut(override val msg: String) extends StorageError(msg)
 
+  final case class CopyOperationFailed(name: String, source: Fs2Path, dest: Fs2Path)
```

Comment:
It would be better to only have domain types in the errors, if we use something other than FS2 then we shouldn't have to change the error here